CMakeLists.txt

```diff
@@ -0,0 +1,144 @@
+cmake_minimum_required(VERSION 3.21)
+
+# Because Qt implementation is a bit buggy,
+# when this is fixed, the setup will be even nicer.
+set(QT_CREATOR_SKIP_CONAN_SETUP ON)
+
+if(APPLE)
+  set(CMAKE_OSX_DEPLOYMENT_TARGET
+      10.15
+      CACHE STRING "Minimum macOS version the build will be able to run on"
+            FORCE)
+endif()
+
+project(
+  JASP
+  VERSION 0.16.2.0 # <major>[.<minor>[.<patch>[.<tweak>]]]
```

Comment:
So this is where we define the version from now on?

Reply:
Yes!